test/integration/targets/dms_replication_subnet_group/tasks/main.yml

```diff
@@ -0,0 +1,144 @@
+---
+
+- name: set connection information for all tasks
+  set_fact:
+    aws_connection_info: &aws_connection_info
```

Comment:
You'll need to use the connection info in each of the test tasks: 
    <<: *aws_connection_info